tests/sirv-cli.js

```diff
@@ -149,4 +149,19 @@ http2('should start a HTTP/2 server with valid args', async () => {
 	}
 });
 
+http2('should have backward compatibility with HTTP/1', async () => {
+	let pems = selfsigned.generate();
+	let key = await utils.write('foobar.key2', pems.private);
+	let cert = await utils.write('foobar.cert2', pems.cert);
+
+	let server = await utils.spawn('--http2', '--key', key, '--cert', cert, '--port', '5001');
```

Comment:
Was there a particular reason for `--port 5001` here?
```suggestion
	let server = await utils.spawn('--http2', '--key', key, '--cert', cert);
```

Reply:
Yeah, I got an `ECONNRESET` error in the tests on an earlier commit without it. https://github.com/lukeed/sirv/runs/921743587#step:8:40

I'm not sure if that was a fluke, but I can try again w/o `--port 5001`.

Reply:
Hmm, I guess the 0a19722 test failure was a fluke? The last commit worked just fine.